src/main/java/seedu/parser/Parser.java

```diff
@@ -2,13 +2,7 @@
 
 import java.util.ArrayList;
 
-import seedu.command.AddCommand;
-import seedu.command.CheckCommand;
-import seedu.command.Command;
-import seedu.command.DeleteCommand;
-import seedu.command.IncorrectCommand;
-import seedu.command.ListCommand;
-import seedu.command.UpdateCommand;
+import seedu.command.*;
```

Comment:
I tried doing this but the CI checks will whine about it. Let's leave it verbose?